Tests/UnitTests/Core/Propagator/StepperTests.cpp

```diff
@@ -207,11 +204,9 @@ BOOST_AUTO_TEST_CASE(eigen_stepper_test) {
   EigenStepper<ConstantBField> es(bField);
 
   // Test the getters
-  BOOST_CHECK_EQUAL(es.position(esState), esState.pos);
-  BOOST_CHECK_EQUAL(es.direction(esState), esState.dir);
-  BOOST_CHECK_EQUAL(es.momentum(esState), esState.p);
-  BOOST_CHECK_EQUAL(es.charge(esState), esState.q);
-  BOOST_CHECK_EQUAL(es.time(esState), esState.t);
+  BOOST_CHECK_EQUAL(es.position(esState), pos);
+  BOOST_CHECK_EQUAL(es.charge(esState), charge);
+  BOOST_CHECK_EQUAL(es.time(esState), time);
```

Comment:
Why are tests for momentum and direction missing?

Reply:
RE-introduced.